apps/opencs/view/doc/runlogsubview.hpp

```diff
@@ -13,7 +13,7 @@ namespace CSVDoc
 
             RunLogSubView (const CSMWorld::UniversalId& id, CSMDoc::Document& document);
 
-            virtual void setEditLock (bool locked);
+            virtual void setEditLock (bool locked) override;
```

Comment:
Shouldn't it be that in this particular case, we drop virtual as well?   The virtual void setEditLock is inherited from SubView, s here, it would be:  void setEditLock (bool locked) override;